sdk/communication/azure-communication-networktraversal/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.1 (2021-08-19)
+## 1.0.0-beta.1 (2021-08-25)
 - Preview release of `azure-communication-networktraversal`.
 
 The first preview of the Azure Communication Relay Client has the following features:
```

Comment:
Dont think this is needed. Everything can sit under the Feartures Added subheading